src/SDKs/Azure.Base/data-plane/Azure.Base/Diagnostics/HttpPipelineEventSource.cs

```diff
@@ -1,79 +1,193 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using Azure.Base.Http;
 using System;
+using System.Collections.Generic;
+using Azure.Base.Http;
 using System.Diagnostics.Tracing;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
 
 // TODO (pri 2): we should log correction/activity
 // TODO (pri 2): we should log exceptions
 namespace Azure.Base.Diagnostics
 {
     // TODO (pri 2): make the type internal
-    [EventSource(Name = SOURCE_NAME)]
+    [EventSource(Name = EventSourceName)]
     internal sealed class HttpPipelineEventSource : EventSource
     {
         // TODO (pri 3): do we want the same source name for all SDk components?
-        const string SOURCE_NAME = "AzureSDK";
+        private const string EventSourceName = "AzureSDK";
 
-        const int LOG_REQUEST = 3;
-        const int LOG_RESPONSE = 4;
-        const int LOG_DELAY = 5;
-        const int LOG_ERROR_RESPONSE = 6;
+        // Maximum event size is 32K but we need to account for other data
+        private const int MaxEventPayloadSize = 31000;
```

Comment:
How did you pick up these const values?

Reply:
Wrongly. I mixed up EventLog (that has 32K limit) and EventSource (~10K limit). 

Last time I wrote EventSource that produces a lot of logs I experimented with payload sized and it seems like even though https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.eventing.eventprovider.writeevent?view=netframework-4.7.2 claims the limit is 64K messages with payloads larger than 10K would often get lost.